Examples/test-suite/ret_by_value.i

```diff
@@ -4,6 +4,12 @@
 
 %warnfilter(SWIGWARN_RUBY_WRONG_NAME) test; /* Ruby, wrong class name */
 
+#ifdef SWIGFORTRAN
+// Treat the struct as a native fortran struct rather than as a class with
```

Comment:
I think you should put all the %fortran_bindc_struct tests into one fortran specific test and not customize these common tests so that it can be shown that the default wrappers work. The ret_by_value_runme.f90 test should be converted to the non-bindc approach to handling struct members. Actually, it isn't clear to me how member variables in a class are handled (when not using %fortran_bindc_struct). Could you add in a section and example on this in the docs as member variable handling is fundamental.

Reply:
Fixed in 791fd0431